lib/reader.js

```diff
@@ -44,5 +45,24 @@ module.exports = async (feeds, options) => {
         })
     );
 
-    return getStream(browserify({ entries, debug: opts.debug }).bundle());
+    const bundler = browserify({ entries, debug: opts.sourceMaps });
+
+    let envifyOptions = { ...process.env, _: 'purge' };
```

Comment:
We set `NODE_ENV` to production on our deployed apps, so either ignore it or explicitly set `"development"`

Reply:
When you say ignore, do you mean remove setting it here at all?

Reply:
```js
const {NODE_ENV, ...env} = process.env

let envifyOptions = { ...env, _: 'purge' };
....
```

Reply:
hmm, wouldn't we want to replace NODE_ENV?

Reply:
Yes, but you only replace it in `if (opts.minify)`. And you set it to `production` as well. So now it's a noop for us.

What about this:

```js
   let envifyOptions = { ...process.env, _: 'purge', NODE_ENV: 'development' };

   if (opts.minify) {
        envifyOptions = { ...envifyOptions, NODE_ENV: 'production' };
    }
```

Reply:
I guess I figured that by including it we are saying "We will respect your NODE_ENV unless you request minification in which case we are going to set it to prod". That seemed legit to me as an override. 

I'm also not against just no special treatment where NODE_ENV is always respected. 

I guess explicitly setting dev and prod is ok but what happens if someone had like a staging env or some such?

Reply:
since where we care about minification our envs will be in prod anyway, wouldn't this be fine?

```js
bundler.transform(envify, { ...process.env, _: 'purge' });
```

Reply:
Again, all of our apps run with `NODE_ENV = 'production'` already, meaning that for FINN the current `if (opts.minify) {` is dead code - it will never modify the variable passed on.

>I'm also not against just no special treatment where NODE_ENV is always respected.

In practice, this will be the behavior for us at FINN.

>since where we care about minification our envs will be in prod anyway, wouldn't this be fine?

I would like some way to opt out of it when working locally. Not _too_ important, though

Reply:
ah yea,
are there situations where always replacing NODE_ENV when developing would be a suck fest?